src/main/java/bern/functionalities/TaskList.java

```diff
@@ -0,0 +1,30 @@
+package bern.functionalities;
+
+import java.util.ArrayList;
+
+import bern.model.Task;
+
+/**
+ * A class to encapsulates all methods TaskList related.
+ */
+public class TaskList {
+    /**
+     * A method to add task into an ArrayList of Tasks.
+     *
+     * @param task The additional task.
+     * @param arListTask The initial ArrayList of Tasks.
+     */
+    public void addTask(Task task, ArrayList<Task> arListTask) {
```

Comment:
The name `arrListTask` suggests that the variable is a task. Consider `taskArrList` instead?